baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/ZeroWarmupRateLimiterTest.java

```diff
@@ -66,6 +66,29 @@ public void should_remove_duration_zero_static_import() {
                 .doTest();
     }
 
+    @Test
+    public void should_remove_long_literal_zero() {
+        fix().addInputLines(
+                        "Test.java",
+                        "import com.google.common.util.concurrent.RateLimiter;",
+                        "import java.util.concurrent.TimeUnit;",
+                        "class Test {",
+                        "  void f() {",
+                        "    RateLimiter.create(10, 0, TimeUnit.SECONDS);",
```

Comment:
Might be helpful to add a test using a long literal `0L`

Reply:
Good idea.